R/tidyverse.R

```diff
@@ -210,7 +210,7 @@ use_tidy_support <- function() {
   use_template(
     "tidy-support.md",
     path(".github", "SUPPORT.md"),
-    data = list(package = project_name())
+    data = list(package = github_repo_spec())
```

Comment:
Could you please rename `package` to repo?

Reply:
I believe this update is outdated based on changes in c32beffe and perhaps earlier commits.